### DISCUSSION TITLE
Switch Storage Account?

### DISCUSSION THREAD

**Discussion by ckimwilshire** [microsoft/DurableFunctionsMonitor#160] [Q&A] [answered]
Hello,

Is there any way to switch between storage accounts within UI, or specify task hubs that are in different storage accounts?

**Comment by scale-tone** [comment, marked as answer]
@ckimwilshire , in VSCode ext you should see all your Task Hubs in all Storage accounts:

<img src="https://github.com/microsoft/DurableFunctionsMonitor/assets/5447190/a836ba72-1367-409f-878b-16656b29793e" width="400px"/>

InProc/Standalone DfMon does supports multiple Storage accounts, via [DFM_ALTERNATIVE_CONNECTION_STRING_XXX](https://github.com/microsoft/DurableFunctionsMonitor/wiki/Config-Settings-Reference) settings. 

Isolated DfMon does not support multiple Storage accounts and is unlikely to support.
